.idea/codeStyles/Project.xml

```diff
@@ -33,31 +33,6 @@
         </value>
```

Comment:
.idea .gradle is not supposed to be committed

Reply:
I have read that codeStyles folder should be committed.

Reply:
Okay